### ISSUE TITLE
Error when updating a dataset via the cli (Unexpected token < in JSON at position 0)

### ISSUE THREAD

**Issue by chrisgorgo** [OpenNeuroOrg/openneuro#861]
```
(node:107519) UnhandledPromiseRejectionWarning: Error: Network error: Unexpected token < in JSON at position 0
    at new ApolloError (/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/apollo-client/bundle.umd.js:85:32)
    at Object.error (/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/apollo-client/bundle.umd.js:916:32)
    at notifySubscription (/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:130:18)
    at onNotify (/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:161:3)
    at SubscriptionObserver.error (/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:220:7)
    at notifySubscription (/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:130:18)
    at onNotify (/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:161:3)
    at SubscriptionObserver.error (/work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/zen-observable/lib/Observable.js:220:7)
    at /work/01329/poldrack/lonestar/software_lonestar5/nodejs/nodejs-8.11.4/lib/node_modules/openneuro-cli/node_modules/apollo-upload-client/lib/main/index.js:156:24
    at <anonymous>
(node:107519) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:107519) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
There is no indication that this was caused by unreliable network connection. I am starting to suspect that those errors are due to the API timing out.

**Comment by nellh** [comment]
Yes, "Unexpected token < in JSON at position 0" is an API timeout.

**Comment by nellh** [comment]
#843 is another example of this.

**Comment by nellh** [comment]
This should be fixed with 2.2

**Comment by TrevorKMDay** [comment]
I'm not sure where to put this, but I had the same issue with 3.0.1 and was able to work around it by:

1. Uploading a single subject to create the dataset.
2. Uploading everyone else
3. Creating a new snapshot, which picked up the new files

(see ds001907)
